string color
string type
Pt2D[4] region
float32[2] center
float32 theta
